### PR TITLE
Fix group

### PR DESCRIPTION
**Pull request by Assyarul**
Current implementation has no indicator when everyone is shown in
the list. There are also some typos for the group related commands.

Let's fix those.

### CODE REVIEW

**Review by lue97 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/AddGroupCommand.java [hunk 1]
```diff
@@ -20,16 +21,17 @@
...
             + "add people into group if they are not in it.  \n"
             + "Parameters: " + PREFIX_NAME + "GROUP_NAME (must be alphanumeric only) "
             + PREFIX_PERSONS + "[INDEX]\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "Close Friends" + " "
             + PREFIX_PERSONS + "1 2 3 4";
 
-    public static final String MESSAGE_ADD_GROUP_SUCCESS = "Added %1$s into group %2$s";
+    public static final String MESSAGE_ADD_GROUP_SUCCESS = "Added into group %1$s";
+    public static final String MESSAGE_ADD_DEFAULT_GROUP_ERROR = "Unable to add to the Default Group, %1$s.";
```
Comment (chesterhow) [inline src/main/java/seedu/address/logic/commands/AddGroupCommand.java]: Can reword the error message to `"There is no need to add people to \"%1$s\" as everyone is included by default."`

File: src/main/java/seedu/address/logic/commands/DeleteGroupCommand.java [hunk 2]
```diff
@@ -19,6 +20,7 @@
             + PREFIX_NAME + "Close Friends";
 
     public static final String MESSAGE_DELETE_GROUP_SUCCESS = "Deleted group %1$s";
+    public static final String MESSAGE_DELETE_DEFAULT_GROUP_ERROR = "Unable to delete the Default Group, %1$s.";
```
Comment (chesterhow) [inline src/main/java/seedu/address/logic/commands/DeleteGroupCommand.java]: Can reword the error message to `"\"%1$s\" cannot be deleted."`

File: src/main/java/seedu/address/logic/commands/ListCommand.java [hunk 3]
```diff
@@ -35,7 +36,9 @@ public ListCommand(Optional<Name> name) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         if (this.name.isEmpty()) {
-            model.setCurrentGroup(null);
+            Group defaultGroup = model.getGroupMap().get(DEFAULT_GROUP_NAME);
+            model.setGroup(defaultGroup.getName(), defaultGroup);
+            model.setCurrentGroup(defaultGroup);
```
Comment (chesterhow) [inline src/main/java/seedu/address/logic/commands/ListCommand.java]: `model.setGroup` will update `currentGroup` already so line 41 is not needed

**Review by chesterhow — APPROVED** [review]
LGTM, just some minor changes and we need to consider cases where another command switches back to the All group

**Pull request merged**